CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ how a consumer would use the library (e.g. adding unit tests, updating documenta
 
 - Bug where an empty destination list in a device's backup set broke creation of DeviceSettings objects for that device.
 
+### Added
+
+- Added new exception `Py42CloudAliasLimitExceeded` to throw if `add_cloud_alias` throws `400` and body contains reason
```

Comment:
`add_cloud_alias` is a function and looks better if ending with `()`

`add_cloud_alias()`

Reply:
Changed.